src/server.ts

```diff
@@ -178,6 +189,9 @@ const startServer = async () => {
       process.exit(1)
     }
     console.log(generateAsciiHeader(config.version))
+    console.log(DateTime.now().toLocaleString(DateTime.DATETIME_HUGE_WITH_SECONDS))
+    console.log('Locale:', Settings.defaultLocale)
+    console.log('Timezone:', Settings.defaultZoneName)
```

Comment:
Useful to print this on server start so we can be completely sure what timezone the server thinks it's in and what the current time is in that timezone.